shared/locales/intl/unitformat.en-US.properties

```diff
@@ -37,3 +37,43 @@ duration-month-narrow[two]    = {{ value }}mths
 duration-month-narrow[few]    = {{ value }}mths
 duration-month-narrow[many]   = {{ value }}mths
 duration-month-narrow[other]  = {{ value }}mths
+
+digital-byte-short            = {[ plural(value) ]}
```

Comment:
I just follow what they're discussing in ecma 402 proposal. My take on the reason for the groupName is to aid in a scenario when two units from different areas have the same name (for example angle second vs. time second).